### PR TITLE
move ModelRun under FeatureExtraction

### PR DESCRIPTION
**Pull request by zekun-li**

### CODE REVIEW

**Review by asaxton — APPROVED** [review]
Looks good to me. But I'm happy to yield to other reviewer.

**Review by dgiesbrecht — APPROVED** [review]

File: ta1/output.py [hunk 1]
```diff
@@ -293,5 +301,40 @@ class Map(BaseModel):
     features: MapFeatureExtractions
     cross_sections: Optional[list[CrossSection]]
 
-    pipelines: list[ModelRun]
+    
     projection_info: ProjectionMeta
+
+
+
+graph = erd.create(Map)
```
Comment (davenquinn) [inline ta1/output.py:309]: I think this diagram creation code is now duplicative of code in https://github.com/DARPA-CRITICALMAAS/schemas/blob/main/scripts/generate-schema.py. This code  (and the imports that it references) should be deleted for a clean merge.

File: ta1/output.py [hunk 2]
```diff
@@ -2,6 +2,11 @@
 from typing import Optional, TypeVar
 from enum import Enum
 
+import erdantic as erd
```
Comment (davenquinn) [inline ta1/output.py:5]: see comment below

**Review by davenquinn — CHANGES_REQUESTED** [review]
Looks good, except that duplicated generation code for `output.png` should be removed.

File: ta1/output.py (hunk 2 above)
Comment (zekun-li) [reply ta1/output.py]: Resolved in the latest commit

**Pull request merged**